### ISSUE TITLE
integration.cloud.clouds.test_ec2 test failures

### ISSUE THREAD

**Issue by Ch3LL** [saltstack/salt#52447]
### Description of Issue/Question
The tests:

```
integration.cloud.clouds.test_ec2.EC2Test.test_win2012r2_winrm
integration.cloud.clouds.test_ec2.EC2Test.test_win2016_psexec
integration.cloud.clouds.test_ec2.EC2Test.test_win2016_winrm
```

are failing on 2019.2 with failure: 

```
Traceback (most recent call last):
  File "/tmp/kitchen/testing/tests/integration/cloud/clouds/test_ec2.py", line 228, in test_win2012r2_psexec
    self._test_instance('ec2-win2012r2-test', debug=True, timeout=TIMEOUT)
  File "/tmp/kitchen/testing/tests/integration/cloud/clouds/test_ec2.py", line 157, in _test_instance
    self.assertIn(ret_str, instance)
AssertionError: u'CLOUD-TEST-MNDTGE:' not found in []
```

Might be directly related to fix in https://github.com/saltstack/salt/issues/52051

**Comment by Ch3LL** [comment]
still failing after merge

**Comment by DmitryKuzmenko** [comment]
The same in 2019.2 with:
integration.cloud.clouds.test_ec2.EC2Test.test_instance
integration.cloud.clouds.test_ec2.EC2Test.test_instance_rename
integration.cloud.clouds.test_ec2.EC2Test.test_win2012r2_psexec
integration.cloud.clouds.test_ec2.EC2Test.test_win2012r2_winrm
integration.cloud.clouds.test_ec2.EC2Test.test_win2016_psexec
integration.cloud.clouds.test_ec2.EC2Test.test_win2016_winrm

https://jenkinsci.saltstack.com/job/2019.2/view/Cloud/job/salt-centos-7-py2-cloud/142/testReport/junit/integration.cloud.clouds.test_ec2/EC2Test/test_instance/

```
Error Message

u'CLOUD-TEST-ADXDQN:' not found in []

Stacktrace

Traceback (most recent call last):
  File "/tmp/kitchen/testing/tests/integration/cloud/clouds/test_ec2.py", line 211, in test_instance
    self._test_instance('ec2-test')
  File "/tmp/kitchen/testing/tests/integration/cloud/clouds/test_ec2.py", line 157, in _test_instance
    self.assertIn(ret_str, instance)
AssertionError: u'CLOUD-TEST-ADXDQN:' not found in []
```

**Comment by Ch3LL** [comment]
will be fixed on 2019.2 after merge forward from 2019.2.1
